tests/object_file_system_test.php

```diff
@@ -291,9 +293,11 @@ public function test_delete_empty_folders(
         }
         $this->filesystem->delete_empty_dirs($testdir);
         foreach ($dirs as $key => $dir) {
-             $this->assertEquals($expectedparentreadable[$key], is_readable($testdir . $dir));
+            $this->assertEquals($expectedparentreadable[$key], is_readable($testdir . $dir));
         }
         $this->assertEquals($expectedgrandparentpathreadable, is_readable($testdir));
+        // clean up.
+        $this->rrmdir($testdir);
```

Comment:
if the issue is `resetAfterTest` is not working I think it is safe to do:
```
shell_exec('rm -rf ' .  $testdir);
```
I've tested it locally(totara:CatalystLearn12-3)

Reply:
Pushed changes to clear filedir after each test.